setup.sh

```diff
@@ -16,7 +16,7 @@ function runDots() {
             echo "Syncing the dev-setup repo to your local machine."
             echo "------------------------------"
             echo ""
-            cd ~ && curl -#L https://github.com/donnemartin/dev-setup/tarball/master | tar -xzv --strip-components 1 --exclude={README.md,LICENSE}
+            ./bootstrap.sh
```

Comment:
I think we could update the echo description and use the `--force` flag to prevent the *double* confirmation prompt "This script may overwrite existing files in your home directory. Are you sure? (y/n)":

```
...
        if [ $ARG == "bootstrap" ] || [ $ARG == "all" ]; then
            echo ""
            echo "------------------------------"
            echo "Syncing the dev-setup dotfiles to your home directory."
            echo "------------------------------"
            echo ""
            ./bootstrap.sh --force
        fi
...
```